src/System.Diagnostics.Process/tests/ProcessThreadTests.cs

```diff
@@ -14,6 +14,7 @@ namespace System.Diagnostics.Tests
     public class ProcessThreadTests : ProcessTestBase
     {
         [Fact]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.UapNotUapAot, "https://github.com/dotnet/corefx/issues/22174")]
```

Comment:
We should use [ActiveIssue] for tests that are disabled against an open issue. Just to keep consistency in the repo :)